OpenShift/helm/nexus-repository-manager/values.yaml

```diff
@@ -36,7 +36,6 @@ nexus:
   dockerPort: 5003
   nexusPort: 8081
   service:
-    type: NodePort
```

Comment:
unused, so removed it.

Reply:
we should just make it 'used'

Reply:
That can be a PR for the future, I think, since we could apply it to charts for both servers.